src/main/resources/templates/index.html

```diff
@@ -5,13 +5,16 @@
 	<meta charset="ISO-8859-1"/>
 	<link rel="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
 	<link rel="stylesheet" type="text/css" href="../static/stylesheet.css" />
-	<title>NeighborhoodAssurance.com</title>
+	<title>Neighborhood Assurance</title>
 	
 </head>
 
 <body>
 	<h1>Welcome to Neighborhood Assurance!</h1>
-	
+	<h3>This program allows users to insert an address and return criminal statistics
+	 based on the neighborhood. This will provide a general idea of crime rate, and a 
+	 statistical analysis if the neighborhood criminal activity has increased or 
+	 decreased over time.</h3>

```

Comment:
This page was pretty empty and I didn't want to write a lot of code BUT one thing that is absolutely necessary on a landing page is some kind of description.  I added the description from your readme here